{"name": "test_valid_auth[standard_user-secret_sauce]", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable", "trace": "self = <tests.test_auth.TestAuth object at 0x0000012C05E145E0>, email = 'standard_user', password = 'secret_sauce'\napp = <pages.application.Application object at 0x0000012C06852B80>\n\n    @allure.story(\"Авторизация\")\n    @allure.severity(\"blocker\")\n    @pytest.mark.parametrize(\n        \"email, password\",\n        (\n                (Users.STANDARD_USERNAME, Users.PASSWORD),\n                (Users.PROBLEM_USERNAME, Users.PASSWORD),\n                (Users.PER_GLITCH_USERNAME, Users.PASSWORD)\n        ),\n    )\n    def test_valid_auth(self, email, password, app):\n        \"\"\"\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        \"\"\"\n        app.open_main_page()\n        app.login.auth(email, password)\n    \n>       assert (app.main_page.logout_button_text() ==\n                ButtonTexts.LOGOUT_BUTTON_TEXT, \"Кнопка Logout не найдена\")\n\ntests\\test_auth.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.main_page.MainPage object at 0x0000012C05EA2E20>\n\n    def logout_button_text(self):\n>       self.logout_button().text()\nE       TypeError: 'str' object is not callable\n\npages\\main_page.py:42: TypeError"}, "description": "\n        1. Открыть страницу\n        2. Ввести валидные значения в поля логина и пароля\n        3. Нажать кнопку LOGIN\n        4. Проверить, что в бургере есть кнопка Logout\n        ", "attachments": [{"name": "screenshot", "source": "0b320e40-1c3b-4103-a7c7-3423d7704a30-attachment.png", "type": "image/png"}, {"name": "log", "source": "17eb1c26-26a7-4554-afbb-83a1e159fca1-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "ffe3ff54-0970-4420-8b73-9f892b33595b-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9cefda04-1580-4d79-9eb3-3ec06d687e4b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "email", "value": "'standard_user'"}, {"name": "password", "value": "'secret_sauce'"}], "start": 1615012528705, "stop": 1615012530065, "uuid": "912b9543-6238-45b6-a18b-fabeca9e0669", "historyId": "7ccc94e307ff0a86994e8a3baf242069", "testCaseId": "52b3ff36bc4dce34c6b761d4e4c54109", "fullName": "tests.test_auth.TestAuth#test_valid_auth", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestAuth"}, {"name": "host", "value": "LAPTOP-Q6599UFH"}, {"name": "thread", "value": "9404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}
